5_databases/week1/database_hmwrk1.sql

```diff
@@ -0,0 +1,55 @@
+-- Active: 1706786442782@@127.0.0.1@3306@hyf_homework
+
+-- SELECT * FROM status;
+-- SELECT * FROM task;
+
+-- 1 Find out how many tasks are in the task table
+SELECT COUNT(*) AmountTask FROM task;
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) AS InvalidDueDate FROM task
+WHERE due_date IS NULL;
+
+-- 3 Find all the tasks that are marked as done
+SELECT id AS TaskID, title AS taskTitle, status_id AS Done
+FROM task
+WHERE status_id = 3;
+
+-- 4 Find all the tasks that are not marked as done
+SELECT id AS TaskID, title AS taskTitle, status_id AS NotDone
+FROM task
+WHERE status_id != 3;
+
```

Comment:
Similar for this one 😊

Reply:
Thank you I implemented changes but somehow couldn't push to Github.